app/presenters/two_factor_options_presenter.rb

```diff
@@ -1,28 +1,18 @@
 class TwoFactorOptionsPresenter
   include ActionView::Helpers::TranslationHelper
 
-  attr_reader :user, :priority_bucket
-
-  def initialize(user_agent:, user: nil, priority_bucket: :default,
+  attr_reader :user
```

Comment:
Nit: Can we keep the newline spacing between the `attr_reader` and constructor?

```suggestion
  attr_reader :user

```